pkg/cmd/build/new.go

```diff
@@ -42,6 +48,24 @@ func NewCmdBuildNew(f *factory.Factory) *cobra.Command {
 	return &cmd
 }
 
+func pipelineResolverPositionArg(args []string, conf *config.Config) pipeline.PipelineResolverFn {
```

Comment:
I think these functions could be better or live somewhere else. This serves as the idea for how to integrate it at least